common/cmake/ALPSEnableEigen.cmake

```diff
@@ -0,0 +1,112 @@
+# enable using Eigen
```

Comment:
This file is directly copied from the eigen distro, right?  In this case we should make a note to that effect in order to satisfy the BSD license . . .

Reply:
Sorry, found it. Please ignore.